src/components/card/z-card-footer/index.tsx

```diff
@@ -26,38 +24,88 @@ export class ZCardFooter {
 
   @State() isOpen: boolean = false;
 
+  @State() allowTooltipAuthors: boolean = false;
+  @State() allowTooltipIsbn: boolean = false;
+
+  private footer?: HTMLElement;
+  private ellipsisAuthors?: HTMLElement;
+  private ellipsisIsbn?: HTMLElement;
+
   @Listen("toggleClick")
   handleToggle(): void {
     this.isOpen = !this.isOpen;
   }
 
+  getTitleAuthors(): string {
+    return this.allowTooltipAuthors ? this.autori : "";
+  }
+
+  getTitleIsbn(): string {
+    return this.allowTooltipIsbn ? this.isbn : "";
+  }
+
+  componentDidLoad() {
+    this.footer.ontransitionend = event => {
+      if (["visibility", "height"].includes(event.propertyName))
+        this.elementsEllipsis();
+    };
+  }
+
+  elementsEllipsis(): void {
+    this.allowTooltipAuthors = this.elementAuthorsHasEllipsis() ? true : false;
+    this.allowTooltipIsbn = this.elementIsbnHasEllipsis() ? true : false;
+  }
+
+  elementAuthorsHasEllipsis(): boolean {
+    return this.ellipsisAuthors.offsetWidth < this.ellipsisAuthors.scrollWidth;
+  }
+
+  elementIsbnHasEllipsis(): boolean {
+    return this.ellipsisIsbn.offsetWidth < this.ellipsisIsbn.scrollWidth;
+  }
+
```

Comment:
Perhaps we can remove ellipsis on ISBN here, given that it is always visible now.